packages/react/README.md

```diff
@@ -150,6 +150,19 @@ import { FaroRoutes } from '@grafana/faro-react';
 </FaroRoutes>;
 ```
 
+### V6 Data Router
```

Comment:
Should this be a sub-heading of the previous section?

Reply:
Oh good catch. 
All routers shoudl be oen heading level below `###Router`.
Fixed the outline.